cloudfoundry/resource_cf_user_provided_service.go

```diff
@@ -171,7 +172,7 @@ func resourceUserProvidedServiceRead(ctx context.Context, d *schema.ResourceData
 
 	userProvidedServiceInstance, _, _, err := session.ClientV3.GetServiceInstanceByNameAndSpace(name, space)
 	if err != nil {
-		if IsErrNotFound(err) {
+		if _, ok := err.(ccerror.ServiceInstanceNotFoundError); ok {
```

Comment:
same CR, `errors.Is(err, ccerror.ServiceInstanceNotFoundError)`

Reply:
Same as above